### PR TITLE
routing 5/5 complete

### PR DESCRIPTION
**Pull request by earl-nicholson**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -1,5 +1,17 @@
...
 import PropTypes from "prop-types"
+import {
+  BrowserRouter as  Router,
+  Route,
+  Switch
+} from 'react-router-dom'
+
+
+
+
```
Comment (egomez-papa) [inline app/javascript/components/App.js:11]: What's with all this whitespace?

File: app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -0,0 +1 @@
+ @import 'bootstrap';
```
Comment (iameddieyayaya) [inline app/assets/stylesheets/application.scss:1]: end files with a new line; formatter should handle this.

File: app/models/user.rb [hunk 3]
```diff
@@ -1,4 +1,5 @@
 class User < ApplicationRecord
+  has_many :locations
   # Include default devise modules. Others available are:
```
Comment (iameddieyayaya) [inline app/models/user.rb:3]: delete unnecessary comments

**Review by iameddieyayaya — COMMENTED** [review]
some subtle changes

**Review by iameddieyayaya — APPROVED** [review]

**Pull request merged**